pandas/tests/frame/test_asof.py

```diff
@@ -106,3 +106,20 @@ def test_all_nans(self):
         result = DataFrame(np.nan, index=[1, 2], columns=['A', 'B']).asof(3)
         expected = Series(np.nan, index=['A', 'B'], name=3)
         tm.assert_series_equal(result, expected)
+
+    # Testing awareness of DataFrame index considering different
+    # UTC and different timezone
+    @pytest.mark.parametrize("stamp,expected",
+              [(Timestamp('2018-01-01 23:22:43.325+00:00'),
```

Comment:
this will have some linting errors

you can format this a bit better by doing
```
@pytest.mark.parameterize(
    "stamp, expected",
        [
          (......),
          (......)])
```